x/blob/types/payforblob_test.go

```diff
@@ -244,6 +245,7 @@ func TestValidateBasic(t *testing.T) {
 				space, code, log := sdkerrors.ABCIInfo(err, false)
 				assert.Equal(t, tt.wantErr.Codespace(), space)
 				assert.Equal(t, tt.wantErr.ABCICode(), code)
+				fmt.Printf("code %v", code)
```

Comment:
I'm also fine with merging this PR and iterating if we have issues open, and this line ofc :smile:

Reply:
Resolved via https://github.com/celestiaorg/celestia-app/pull/2145/commits/6e4c660974f2933c7a8e2923638dfa1d71a862a4